api/types.proto

```diff
@@ -290,6 +290,16 @@ message UpdateConfig {
 	// reach the RUNNING state. In the future, there will be configuration
 	// to define what is treated as a failure (see #486 for a proposal).
 	FailureAction failure_action = 3;
+
+	// RolloutStrategy controls the order of operations when rolling out an
+	// updated task. Either the old task is shut down before the new task is
+	// started, or the new task is started before the old task is shut down.
+	enum RolloutStrategy {
```

Comment:
How about calling this `RolloutStartOrder`? RolloutStrategy might be too general, from my point of view, `parallelism` and `delay` are also part of RolloutStrategy.